awscrt/mqtt_request_response.py

```diff
@@ -64,6 +150,9 @@ class RequestResponseOperationOptions:
     correlation_token: 'Optional[str]' = None
 
     def validate(self):
+        """
+        Stringently type-checks an instance's field values.
+        """
         assert isinstance(self.subscription_topic_filters, collections.abc.Sequence)
```

Comment:
out of scope of this specific review but could we use `from collections.abc import Sequence` in imports and use `Sequence` directly? I'm also getting "Sequence is not defined" in the areas of this file where `Sequence` is being used directly despite `import collections.abc` being up top.